test.js

```diff
@@ -1,247 +1,268 @@
-var raf = require('./')
-var tape = require('tape')
-var os = require('os')
-var path = require('path')
-var fs = require('fs')
-var mkdirp = require('mkdirp')
-var isWin = process.platform === 'win32'
+const test = require('brittle')
```

Comment:
what is the motivation for moving from `tape` to `brittle`? Maybe we could consider the new experimental test runner from node core https://github.com/nodejs/node-core-test, in hope that we'll eventually consume it from stdlib?

Reply:
Brittle is overall solid and the rest of the ecosystem, including Hypercore and Hyperswarm, have mostly moved to Brittle as well.

Reply:
Thanks!

Reply:
Pleasure!